EpsilonDashboardSetup.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+echo 'deb http://www.rabbitmq.com/debian/ testing main' | sudo tee /etc/apt/sources.list.d/rabbitmq.list && sudo apt-get update && sudo apt-get install rabbitmq-server
+sudo apt-get install cmake libboost-dev openssl libssl-dev libblkid-dev e2fslibs-dev libboost-all-dev libaudit-dev software-properties-common build-essential mesa-common-dev libgl1-mesa-dev
+
+cd /tmp/
+if [ ls /usr/local/librabbitmq.* 1> /dev/null 2>&1 ];
```

Comment:
Don't need the square bracket for `ls` statements:
https://stackoverflow.com/questions/8934012/when-square-brackets-are-required-in-bash-if-statement

Same for the one below